backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Accessibility {
+  id           String        @id() @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  createdAt    DateTime      @default(now()) @map("created_at") @db.Timestamp(6)
```

Comment:
createdAt / updatedAt / id in core are optional, just want to confirm this is an intentional change

Reply:
good question, this is really (almost strictly) for how things are stored in the db, if create/update has those as optional they are still defaulted/required in the database. 

That optionality will be addressed when we get to the create/update endpoints